docs/articles/expensify-classic/getting-started/create-a-company-workspace.md

```diff
@@ -0,0 +1,171 @@
+              </ul>
+       </ul>
+   <li>Within 1-2 business days, Expensify will send three test transactions to your bank account that you’ll enter into Expensify to validate your bank account by either:</li>  
+       <ul>
+           <li>Clicking the validate task from Concierge on your Home page.</li>
+           <li>Going to Settings > Account > Payments and clicking <b>Enter test transactions</b>.</li>
+       </ul>
+</ol>
+
+{% include info.html %}
+If after two business days you do not see these test transactions, click the green chat bubble in the right corner to get support from Concierge.
+{% include end-info.html %}
+
+# 4. Connect your accounting system
+
+If you use an external accounting system like QuickBooks, you can link it with Expensify to help you import accounting data, code expenses, and more. 
+
+To add an accounting system integration,
+<ol type="a">
+   <li>Hover over Settings, then click <b>Workspaces</b>.</li>
+   <li>Click the <b>Group</b> tab on the left.</li>  
+   <li>Click the desired workspace name.</li>  
+   <li>Click the <b>Connections</b> tab.</li>  
+   <li>Under Accounting Integrations, click the name of your accounting system, select the “Connect to…” option, and click the related button.</li>  
+   <li>Depending on the integration you selected, you’ll either be prompted with a login screen for the accounting system or additional steps for how to proceed.</li>
+</ol>
+  
+For a walkthrough for how to set up a specific accounting system, visit our [Integrations](https://help.expensify.com/expensify-classic/hubs/integrations/) articles.  
+
+# 5. Set approval rules
+
+Determine the basic guidelines that apply to all submitted expenses. If a submitted expense does not meet these rules, it will be flagged as a violation. You can set rules for expenses, per diem, travel, and reports. 
+
+<ol type="a">
+   <li>Hover over Settings, then click <b>Workspaces</b>.</li>
+   <li>Click the <b>Group</b> tab on the left.</li>  
+   <li>Click the desired workspace name.</li>  
+   <li>Click the <b>Expenses</b> tab and set the desired rules.</li>  
+       <ul>
+           <li>Determine if expense violations will be enforced. If enabled, expenses that exceed the set number of days or amount will be flagged as an expense violation.</li>
+           <li>Determine how cash expenses are treated.</li> 
+           <li>Determine if expenses can be re-billed to someone else as an invoice.</li> 
+           <li>Determine if eRecipts can be submitted as proof of an expense.</li> 
+         
+           {% include info.html %}
+           If eReceipts are enabled, imported credit card expenses of $75 USD or less will be tracked automatically—no paper receipt is necessary. 
+
+           eReceipts meet IRS documentation requirements as per Publication 463; However, the IRS will not accept an eReceipt for lodging purchases (for example, hotel expenses will require a paper receipt).
+           {% include end-info.html %}
```

Comment:
@ren-jones this is weird but the level indentation makes jekyll consider it a code block and the formatting doesn't look good.

Could you please remove all indentation for this block.
```
{% include info.html %}
If eReceipts are enabled, imported credit card expenses of $75 USD or less will be tracked automatically—no paper receipt is necessary. 

eReceipts meet IRS documentation requirements as per Publication 463; However, the IRS will not accept an eReceipt for lodging purchases (for example, hotel expenses will require a paper receipt).
{% include end-info.html %}
```

Reply:
@ren-jones it will indent